Makefile

```
@@ -8,6 +8,11 @@ STABLE_VER := $(shell cat molecule/shared/stable.ver)
 SDBIN := $(SDROOT)/securedrop/bin
 DEVSHELL := $(SDBIN)/dev-shell
 
+ifdef $(USE_PODMAN)
```

This didn't work for me, per https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html I think it needs to be the name of the variable, not the value, so:

```suggestion
ifdef USE_PODMAN
```

and that did work for me.

---

Added in e0c1f3783984c579e55d4e7167a37e88d95890e5.